codalab/apps/web/templates/web/my/datasets.html

```diff
@@ -50,15 +51,21 @@
                                             <td>{{ dataset.key }}</td>
                                             <td>{{ dataset.description }}</td>
                                             <td>{{ dataset.type }}</td>
+                                            <td>{{ dataset.is_public|default:"" }}</td>
                                             <td>
-                                                <a href="{% url 'datasets_download' dataset_key=dataset.key %}" class="button btn btn-default btn-sm">Download</a>
+                                                <a href="{% url 'datasets_download' dataset_key=dataset.key %}" class="button btn btn-primary btn-sm">Download</a>
+                                                {% if dataset.is_public %}
+                                                    <a href="{% url 'my_datasets_publish' dataset_key=dataset.key %}" class="button btn btn-default btn-sm publish">Unpublish</a>
```

Comment:
Should probably add some kind of `if(confirm('You sure?'))` to this. So at least we can say "we warned you" if people publish a dataset and shoot their competition in the foot.

Reply:
added